### ISSUE TITLE
adding early Linux history?

### ISSUE THREAD

**Issue by arndb** [bootlin/elixir#142]
I wonder if you could integrate support for the https://archive.org/details/git-history-of-linux tree in addition to the normal kernel repository, this would bring in all releases from v0.1 to v2.6.11.

I don't know if that requires any code changes or just merging in all the tags from that tree before generating the output.

**Comment by MaximeChretien** [comment]
Hi,

This should be added soon, I still have to do some tests to make sure everything works fine before sending my changes.

I used [this](https://git.kernel.org/pub/scm/linux/kernel/git/history/history.git) tree because all versions are tagged here and I made some modifications so it include only the versions not already present in Elixir. 
See : https://github.com/MaximeChretien/linux-history

Some changes were needed to list the tags in the right order but I figured out how to do that.
